### PR TITLE
Renaming Fetch_comments(user) to comments_to_be_read(user)

### LINKED ISSUE #6814: Rename fetch_comments method
`fetch_comments` method should be named something more descriptive, like `comments_viewable_by(user)` or `comments_approved_for(user)`!

_Originally posted by @jywarren in https://github.com/publiclab/plots2/pull/6718#issuecomment-555569444_

The method is here 
https://github.com/publiclab/plots2/blob/master/app/models/node.rb#L1063-L1071

And you need to replace on the new method these files and lines

1. https://github.com/publiclab/plots2/blob/master/app/views/questions/show.html.erb#L58
2. https://github.com/publiclab/plots2/blob/master/app/views/questions/_new_question.html.erb#L71

3. https://github.com/publiclab/plots2/blob/master/app/views//notes/_comments.html.erb#L3

4. https://github.com/publiclab/plots2/blob/master/app/views/dashboard/_node_comment.html.erb#L6

5. https://github.com/publiclab/plots2/blob/master/app/views/dashboard/_node_meta.html.erb#L18

### PR DESCRIPTION
**Pull request by Amith1206**
Screenshot (118)
Screenshot (119)
Screenshot (120)
Screenshot (121)

Fixes #6814  (<=== Add issue number here)

### CODE REVIEW

**Comment by Amith1206** [comment]
My twitter handle is https://twitter.com/AmithTony?s=03

**Comment by cesswairimu** [comment]
Hello @Amith1206, thanks for opening this.. I will add the checklists in the description back so that we can get a little organized. About the screenshot, we require a screenshot of the web page not the actual code, sorry that can be confusing. in this case we would want a screenshot of `/wiki/topic` where we have comments..let me know if you need any help

File: app/models/node.rb [hunk 1]
```diff
@@ -1070,7 +1070,7 @@ def draft_url(base_url)
     base_url + '/notes/show/' + nid.to_s + '/' + token
   end
 
-  def fetch_comments(user)
+  def comments_to_be_read(user)
```
Comment (cesswairimu) [inline app/models/node.rb]: My thinking was `comments_viewable_by(user) ` was a more descriptive method name. What do you think? Thanks again

**Comment by cesswairimu** [comment]
There a couple of failing tests on Travis too...that is because the method is still present in some tests...You can do a search and replace on all the tests where `fetch_comments` is being used and replace with the new method, then we should be good with Travis

File: app/models/node.rb (hunk 1 above)
Comment (Amith1206) [reply app/models/node.rb]: Yes your rit @cesswairimu .......

Comment (cesswairimu) [reply app/models/node.rb]: cool, ping me if you need any help. Thanks

**Comment by cesswairimu** [comment]
Hello @Amith1206, just cecking if you need any of my help here. Thanks

**Comment by cesswairimu** [comment]
This PR has been open for a long time and no activity/ reviews requested has not been addressed. We understand you might be busy and engaged on other things. I am closing this for now...please  feel free to reopen if you get some time and would like to finish this. Rem to check if its still available before you reopen. Thanks

**Pull request closed without merging**